.github/workflows/client-credential-benchmark.yml

```diff
@@ -0,0 +1,46 @@
+name: msal-node client-credential Regression Test
+on:
+    # Allows manual triggering of workflow
+    workflow_dispatch:
+
+    pull_request:
+        types: [opened, reopened, synchronize, ready_for_review]
+        paths:
+            - "regression-tests/msal-node/client-credential/*"
+            - "lib/msal-node/**/*"
+            - "lib/msal-common/**/*"
+            - "!**.md"
+            - ".github/workflows/client-credential.yml"
+            
+    merge_group:
+        types: [checks_requested]
+
+jobs:
+  benchmark:
+    name: Run msal-node client-credential Regression Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+      - name: Run benchmark
+        run: cd regression-tests/msal-node/client-credential && npm install && node index.js | tee output.txt
+
+      - name: Download previous benchmark data
+        uses: actions/cache@v3
+        with:
+          path: ./cache
+          key: ${{ runner.os }}-benchmark
+
+      - name: Store benchmark result
+        uses: benchmark-action/github-action-benchmark@v1
+        with:
+          name: msal-node client-credential Regression Test
+          tool: 'benchmarkjs'
+          output-file-path: regression-tests/msal-node/client-credential/output.txt
+          external-data-json-path: ./cache/client-credential-benchmark-data.json
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          # Show alert with commit comment on detecting possible performance regression
+          alert-threshold: '110%'
```

Comment:
Out of curiosity, does `110%` threshold produce any false positive? MSAL .Net and Python observe higher deviation than this.

BTW, I wanted to find the answer by myself, but [the diagram page](https://azuread.github.io/microsoft-authentication-library-for-js/dev/bench/) somehow is empty right now. Is that a problem?

Reply:
I have yet to see any false positives. Like discussed previously, that may change when the tests are run on the different GitHub Actions runners. The diagram page is empty because this hasn't been merged in yet. The tests will run and the results will be populated in the diagram every time code is merged into msal-node or a change is made to one of the regression test workflows.

Reply:
> The diagram page is empty because this hasn't been merged in yet

Hmm. In my experience, the diagram showed up long before this PR being merged. It showed up when the perf PR was created the first time (after we manually create a dummy gh-pages branch as part of bootstrapping, that is). It is updated every time we added commits into this PR.

Anyway. Since you have merged in this PR just now, let's see how the diagram and false positive (if any) go.

Reply:
I deleted all the test data when I was done finalizing the code since I was purposefully adding/removing regressions to see how the data would plot in the diagram.